OMGVA PoS/appsettings.json

```diff
@@ -7,7 +7,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "LocalDatabase": "Server=(localdb)\\MSSQLLocalDB;Database=OMGVAPOS;Trusted_Connection=True;"
+    "LocalDatabase": "Server=DESKTOP-3IVF1HD\\OMGVADB;Database=OMGVADB;Trusted_Connection=True;TrustServerCertificate=True"
```

Comment:
Inclusion of this file to gitignore did not work. You also need to stop tracking the file. Not sure how to do it though. Also, are you sure this file will not be changed by someone and then need to be updated?